SU2_CFD/src/drivers/CDiscAdjSinglezoneDriver.cpp

```diff
@@ -270,9 +271,11 @@ void CDiscAdjSinglezoneDriver::SetAdj_ObjFunction(){
   unsigned long IterAvg_Obj = config->GetIter_Avg_Objective();
   su2double seeding = 1.0;
 
+  CWindowingTools windowEvaluator = CWindowingTools();
+
   if (time_stepping){
     if (TimeIter < IterAvg_Obj){
-      seeding = 1.0/((su2double)IterAvg_Obj);
+      seeding = windowEvaluator.GetWndWeight(config->GetKindWindow(),TimeIter, IterAvg_Obj-1)/ ((su2double)IterAvg_Obj);
```

Comment:
Don't use "old style" casts (compiler warning) use static_cast or su2double(IterAvg_Obj)

Reply:
Fixed it :)